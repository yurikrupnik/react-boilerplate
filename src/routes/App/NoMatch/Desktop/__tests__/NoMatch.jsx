import React from 'react';
import { StaticRouter, Route } from 'react-router-dom';
import Component from '../index';

const {
    it,
    expect,
    mount
} = global;

it('renders NoMatch for Desktop', () => {
    const wrapper = mount((
        <StaticRouter context={{}}>
            <Route component={Component} />
        </StaticRouter>
    ));
    expect(wrapper).toMatchSnapshot();
});
