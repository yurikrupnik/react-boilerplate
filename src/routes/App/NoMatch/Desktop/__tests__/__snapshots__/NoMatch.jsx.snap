// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders NoMatch for Desktop 1`] = `
<StaticRouter
  basename=""
  context={Object {}}
  location="/"
>
  <Router
    history={
      Object {
        "action": "POP",
        "block": [Function],
        "createHref": [Function],
        "go": [Function],
        "goBack": [Function],
        "goForward": [Function],
        "listen": [Function],
        "location": Object {
          "hash": "",
          "pathname": "/",
          "search": "",
          "state": undefined,
        },
        "push": [Function],
        "replace": [Function],
      }
    }
  >
    <Route
      component={[Function]}
    >
      <NoMatchDesktop
        history={
          Object {
            "action": "POP",
            "block": [Function],
            "createHref": [Function],
            "go": [Function],
            "goBack": [Function],
            "goForward": [Function],
            "listen": [Function],
            "location": Object {
              "hash": "",
              "pathname": "/",
              "search": "",
              "state": undefined,
            },
            "push": [Function],
            "replace": [Function],
          }
        }
        location={
          Object {
            "hash": "",
            "pathname": "/",
            "search": "",
            "state": undefined,
          }
        }
        match={
          Object {
            "isExact": true,
            "params": Object {},
            "path": "/",
            "url": "/",
          }
        }
        staticContext={Object {}}
      >
        <div>
          <h3>
            destop
          </h3>
          <h3>
            No match for
            <code>
              /
            </code>
          </h3>
        </div>
      </NoMatchDesktop>
    </Route>
  </Router>
</StaticRouter>
`;
